File: Proptimo/Backend/Proptimo/Core/Proptimo.Domain/Entities/RealEstateTypeFeatureValue.cs
using Proptimo.Domain.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proptimo.Domain.Entities
{
    public class RealEstateTypeFeatureValue : BaseEntity
    {
        public int? ValueInt { get; set; }
        public decimal? ValueDecimal { get; set; }
        public bool? ValueBool { get; set; }
        public string? ValueString { get; set; }
        public DateTime? ValueDate { get; set; }

        public string RealEstateId { get; set; }
        public RealEstate RealEstate { get; set; }

        public string RealEstateTypeFeatureId { get; set; }
        public RealEstateTypeFeature RealEstateTypeFeature { get; set; }


    }
}

File: Proptimo/Frontend/proptimo/src/features/api/types/realEstateTypeFeatureValue.ts
export interface RealEstateTypeFeatureValue {
    id: string;
    valueInt?: number;
    valueDecimal?: number;
    valueBool?: boolean;
    valueString?: string;
    valueDate?: string;
    realEstateId: string;
    realEstateTypeFeatureId: string;
}

export interface CreateRealEstateTypeFeatureValueRequest {
    commands: RealEstateTypeFeatureValue[];
}


export interface GetRealEstateTypeFeatureValuesByRealEstateIdResponse {
    data: RealEstateTypeFeatureValue[];
    total: number;
}

export interface UpdateRealEstateTypeFeatureValueRequest {
    commands: RealEstateTypeFeatureValue[];
}

